tests/test_multiline.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from mssqlcli.mssqlbuffer import _is_complete
+
+
+class TestMssqlCliMultiline:
+    testdata = [
+        (None, False),
+        ('', False),
+        ('select 1 /* open comment!\ngo', False),
+        ('select 1\ngo -- another comment', True),
+        ('select 1; select 2, "open quote: go', False),
+        ('select 1; GO', False),
+        ('select 1\n select 2;\ngo', True),
+        ('select 1;', False),
+        ('select 1 go', False)
```

Comment:
okay what happens when the like has more than on go - go go

Reply:
Interestingly, for sqlcmd it fails with: `Sqlcmd: Error: Syntax error at line 2 near command 'go'.`

I added a test case which successfully fails.